resources/assets/sass/regions/_home.scss

```diff
@@ -0,0 +1,284 @@
+.r-home {
+
+	background: #ffffff;
+}
+
+	// Destinations
+
+	.r-home__destinations {
+
+		margin-top: -48px;
+		position: relative;
+		z-index: 2;
+	}
+
+		.r-home__destinations-wrap {
+
+			@include wrap('narrow');
+		}
+
+			.r-home__destinations-action {
+
+				padding: 24px 0;
+			}
+
+	// Ablot
+
+	.r-home__about {}
+
+		.r-home__about-wrap {
+
+			@include wrap('narrow');
+		}
+
+	// Forum
+
+	.r-home__forum {
+
+		padding: 48px 0;
+		background: $color-fill-neutral-light;
+
+		@include respond-to(tablet) {
+
+			padding: 48px 0 192px;
+		}
+	}
+
+		.r-home__forum-wrap {
+
+			@include wrap('narrow');
+		}
+
+			.r-home__forum-title {
+
+				padding-bottom: 24px;
+
+				@include respond-to(tablet) {
+
+					padding-bottom: 36px;
+				}
+			}
+
+			.r-home__forum-column {
+
+				&.m-first {
+
+					padding-bottom: 48px;
+
+					@include respond-to(tablet) {
+
+						@include span(4 of 24);
+
+						padding-bottom: 0;
+					}
+				}
+
+				&.m-last {
+
+					@include respond-to(tablet) {
+
+						@include span(18.5 of 24 at 2.5);
+					}
+				}
+			}
+
+	// News
+
+	.r-home__news {
+
+		padding: 48px 0;
+
+		@include respond-to(tablet) {
+
+			margin-top: -96px;
+			padding: 0 0 72px;
+		}
+	}
+
+		.r-home__news-wrap {
+
+			@include wrap('narrow');
+		}
+
+			.r-home__news-title {
+
+				padding-bottom: 24px;
+
+				@include respond-to(tablet) {
+
+					padding-bottom: 36px;
+				}
+			}
+
+			.r-home__news-column {
+
+				&.m-first {
+
+					@include respond-to(tablet) {
+
+						@include span(7 of 24);
+
+						padding-top: 48px;
+					}
+				}
+
+				&.m-last {
+
+					padding-top: 48px;
+
+					@include respond-to(tablet) {
+
+						@include span(15 of 24 at 3);
+
+						padding-top: 0;
+					}
+				}
+			}
+
+				.r-home__news-block-wrap {
+
+					@include clearfix;
+				}
+
+					.r-home__news-block {
```

Comment:
Line should be indented with spaces, not tabs<br>Selector `r-home__news-block` should be written in lowercase with hyphens